mail/common/src/test/java/com/fsck/k9/helper/MailToTest.java

```diff
@@ -187,4 +188,52 @@ public void testCaseInsensitiveParamWrapper_withoutQueryParameters() {
 
         assertThat(Collections.<String>emptyList(), is(result));
     }
-}
+
+    @Test
+    public void testGetInReplyTo_singleMessageId() {
+        Uri uri = Uri.parse("mailto:?in-reply-to=<7C72B202-73F3@somewhere>");
```

Comment:
By not encoding `<` as `%3C` and `>` as `%3E` you're relying on `Uri.parse()` accepting a URI with an invalid syntax. I don't mind having one such test, but it should be spelled out in the method name what is being tested.
For all other tests please use `%3C` and `%3E`.